src/accessory.ts

```diff
@@ -128,7 +128,7 @@ export default class RoombaAccessory implements AccessoryPlugin {
         this.robotpwd = config.robotpwd;
         this.ipaddress = config.ipaddress;
         this.firmware = "N/A";
-        this.noDockOnStop = config.noDockOnStop;
+        this.dockOnStop = config.dockOnStop;
```

Comment:
Does this work? I thought we needed to manually introduce the default of it being `true` like:

```typescript
this.dockOnStop = config.dockOnStop !== undefined ? config.dockOnStop : true;
```

But does Homebridge enforce the schema?

Reply:
Good question. Since it has a default value in the schema, anyone who installs it will get that value. It's currently not a required field in the scheme but we could make it so and that should cover most scenarios where it might be undefined by being left out of the config altogether.

Reply:
@rcoletti116 I don't want to make it required... have you tested that having a default value in the schema means that the plugin receives that default value if it's missing from the schema? That was my question.

Reply:
If it is missing from the config, the schema default does not pass through here. Added requested change to address this.